source/book/include/marlin/book/BookStore.h

```diff
@@ -126,20 +130,33 @@ namespace marlin {
        *  @brief get Entry from key.
        *  @throw BookStoreException key not exist in Store.
        */
-      details::Entry &get( const EntryKey &key ) { return get( key.hash ); }
+      details::Entry &get( const EntryKey &key ) const { return get( key.idx ); }
```

Comment:
fixed added const modifier,
rename key.hash to idx, because it is an id and not an hash

Reply:
If this is const why don't you return a const reference to the `details::Entry` ?

Reply:
is for internal usage (private), because the BookStore only handles pointer to Entry it is no problem, I use it in the `remove(key)` function, like: `get(key).delete()`, which is with const Entry not possible

Reply:
If it returns a reference without const, you should remove the const from the function signature